app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The function returns immediately upon encountering a `VaccineError`, which means it will not check the rest of the friends. If the intention is to check all friends before returning a message, consider accumulating messages or using a different approach to handle this.